src/JenkinDemo/Jenkins.java

```diff
@@ -24,7 +24,7 @@ public void testLogin() throws Exception {
 		
 		System.out.println(title);
 		
-		Assert.assertTrue(title.equals("Hello World"));
+		Assert.assertTrue(title.equals("Hello World...."));
```

Comment:
No need of 4 dots. Change to 3.
```suggestion
		Assert.assertTrue(title.equals("Hello World..."));
```